packages/backend-common/src/middleware/requestLoggingHandler.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { RequestHandler } from 'express';
+import { Logger } from 'winston';
+import morgan from 'morgan';
```

Comment:
Possible that we could want https://www.npmjs.com/package/express-winston instead, but tbh morgan seems fine.

Reply:
That does look great, let's iterate!

Reply:
we've been trying to do this ~now